### PR TITLE
add Emmet support on HTML editor, resolves #13

### LINKED ISSUE #13: Emmet in editor (HTML + JS)
Hi all! @voronianski, thank you for this project!
I want to use the one, but it doesn't support Emmet. It's very-very sadly. Can someone do it?

### PR DESCRIPTION
**Pull request by minwe**
- ACE Editor emmet extension don't work with latest emmet (API changed), so I get an old version from https://github.com/cloud9ide/emmet-core. It also can't bundle into `Editors.jsx`, Webpack throw an error, I just put it oin `assets/vendor`.
- fixes Webpack dev config, also

### CODE REVIEW

File: src/components/Editors.jsx [hunk 1]
```diff
@@ -44,7 +45,7 @@ class Editors extends React.Component {
     }
 
     render() {
-        const { active, code, html, json, error, tabSize } = this.props;
+        const { active, code, html, json, error, tabSize} = this.props;
```
Comment (gsklee) [inline src/components/Editors.jsx]: @minwe Would be better if the whitespace is retained.

File: src/components/Editors.jsx [hunk 2]
```diff
@@ -76,7 +77,8 @@ class Editors extends React.Component {
                         height="auto"
                         onChange={this.handleChange('onHTMLChange')}
                         showPrintMargin={false}
-                        editorProps={{$blockScrolling: Infinity}}
+                        editorProps={{$blockScrolling: Infinity, enableEmmet: true}}
+                        setOptions={{enableEmmet: true}}
```
Comment (voronianski) [inline src/components/Editors.jsx]: @minwe I was thinking of making emmet optional, what do you think?

**Comment by voronianski** [comment]
@minwe thanks a lot! the only thing we should consider is to make emmet enabling optional.

File: src/components/Editors.jsx (hunk 2 above)
Comment (minwe) [inline src/components/Editors.jsx]: `enableEmmet` option on this line is unnecessary. It's my mistake.

Do you mean add a option on `Actions` menu, or let `setOptions` as a prop of Editors component?

Comment (voronianski) [inline src/components/Editors.jsx]: I meant `Actions` menu.

**Comment by minwe** [comment]
@voronianski 

In my opinion, Emmet option in `Actions` menu is unnecessary. 

Enable Emmet has no effect on users who don't use emmet.

The PR is just FIY, you can make a final decision.

**Comment by voronianski** [comment]
@minwe thanks, I've added emmet support via https://github.com/voronianski/esnextbin/commit/3622a867c1f5be89e814141e585838a84ca9034e as your PR had conflicts.

**Pull request closed without merging**